commands/init.go

```diff
@@ -45,13 +45,14 @@ func transformInitArgs(args *Args) error {
 		err    error
 	)
 
-	if args.IsParamsEmpty() {
+	words := args.Words()
```

Comment:
Yes, in the invocation `git init --template foo`, `foo` is a "word" since it doesn't start with "-" but is not the directory argument.